statsmodels/formula/api.py

```diff
@@ -1,33 +1,28 @@
-from statsmodels.regression.linear_model import GLS
-gls = GLS.from_formula
-from statsmodels.regression.linear_model import WLS
-wls = WLS.from_formula
-from statsmodels.regression.linear_model import OLS
-ols = OLS.from_formula
-from statsmodels.regression.linear_model import GLSAR
-glsar = GLSAR.from_formula
-from statsmodels.regression.mixed_linear_model import MixedLM
-mixedlm = MixedLM.from_formula
-from statsmodels.genmod.generalized_linear_model import GLM
-glm = GLM.from_formula
-from statsmodels.robust.robust_linear_model import RLM
-rlm = RLM.from_formula
-from statsmodels.discrete.discrete_model import MNLogit
-mnlogit = MNLogit.from_formula
-from statsmodels.discrete.discrete_model import Logit
-logit = Logit.from_formula
-from statsmodels.discrete.discrete_model import Probit
-probit = Probit.from_formula
-from statsmodels.discrete.discrete_model import Poisson
-poisson = Poisson.from_formula
-from statsmodels.discrete.discrete_model import NegativeBinomial
-negativebinomial = NegativeBinomial.from_formula
-from statsmodels.regression.quantile_regression import QuantReg
-quantreg = QuantReg.from_formula
-from statsmodels.duration.hazard_regression import PHReg
-phreg = PHReg.from_formula
-from statsmodels.genmod.generalized_estimating_equations import (GEE,
-     OrdinalGEE, NominalGEE)
-gee = GEE.from_formula
-ordinal_gee = OrdinalGEE.from_formula
-nominal_gee = NominalGEE.from_formula
+import statsmodels.regression.linear_model as lm
+import statsmodels.discrete.discrete_model as dm
+import statsmodels.regression.mixed_linear_model as mlm
+import statsmodels.genmod.generalized_linear_model as glm
+import statsmodels.robust.robust_linear_model as roblm
+import statsmodels.regression.quantile_regression as qr
+import statsmodels.duration.hazard_regression as hr
+import statsmodels.genmod.generalized_estimating_equations as gee
+
+gls = lm.GLS.from_formula
+wls = lm.WLS.from_formula
+ols = lm.OLS.from_formula
+glsar = lm.GLSAR.from_formula
+mixedlm = mlm.MixedLM.from_formula
+glm = glm.GLM.from_formula
+rlm = roblm.RLM.from_formula
+mnlogit = dm.MNLogit.from_formula
+logit = dm.Logit.from_formula
+probit = dm.Probit.from_formula
+poisson = dm.Poisson.from_formula
+negativebinomial = dm.NegativeBinomial.from_formula
+quantreg = qr.QuantReg.from_formula
+phreg = hr.PHReg.from_formula
+gee = gee.GEE.from_formula
+ordinal_gee = gee.OrdinalGEE.from_formula
```

Comment:
See Travis log.  At this point “gee” no longer refers to the module, so this raises.